src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/instance_blackrock_depths.cpp

```diff
@@ -8,7 +8,8 @@
 #include "InstanceScript.h"
 #include "ScriptMgr.h"
 
-#define TIMER_TOMBOFTHESEVEN    15000
+#define TIMER_TOMBOFTHESEVEN    30000
```

Comment:
this `TIMER_TOMBOFTHESEVEN` is still used in the project? If not you can remove it

Reply:
Yes, it's used line 606 in same file.
I had to split the timers into 2: one before the event (after having talked to the npc), and one in between the mobs. Before this it was 1 timer for all of it. So everything was on a 15s delay. 15s after talking to the npc the mob would attack, and then every 15s a mob. Now it's 5 and then every 30.
15s might have "still felt okay", but 30 is way too far into the "okay guys let's leave this is not happening" field that it had to be split. It's also more accurate.